### ISSUE TITLE
www-client/ladybird: Filesize does not match recorded size

### ISSUE THREAD

**Issue by TheLukeGuy** [SFTtech/gentoo-overlay#8]
Hi! `www-client/ladybird-9999` fails to emerge with the following:

```
>>> Downloading 'https://raw.githubusercontent.com/publicsuffix/list/master/public_suffix_list.dat'
--2024-06-26 18:06:35--  https://raw.githubusercontent.com/publicsuffix/list/master/public_suffix_list.dat
Resolving raw.githubusercontent.com... 185.199.109.133, 185.199.110.133, 185.199.111.133, ...
Connecting to raw.githubusercontent.com|185.199.109.133|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 315082 (308K) [text/plain]
Saving to: ‘/var/cache/distfiles/ladybird-9999-public_suffix_list.dat.__download__’

          /var/cach   0%[                    ]       0  --.-KB/s               /var/cache/distfile 100%[===================>] 307.70K  1.83MB/s    in 0.2s    

2024-06-26 18:06:35 (1.83 MB/s) - ‘/var/cache/distfiles/ladybird-9999-public_suffix_list.dat.__download__’ saved [315082/315082]

!!! Fetched file: ladybird-9999-public_suffix_list.dat VERIFY FAILED!
!!! Reason: Filesize does not match recorded size
!!! Got:      315082
!!! Expected: 314660
Refetching... File renamed to '/var/cache/distfiles/ladybird-9999-public_suffix_list.dat._checksum_failure_.cevvk0zc'

!!! Couldn't download 'ladybird-9999-public_suffix_list.dat'. Aborting.
```

Presumably it's caused by the public suffix list updating?

**Comment by TheJJ** [comment]
ah yes, apparently. hm, but I don't see an obvious way for getting a specific file version, or is there?

**Comment by TheLukeGuy** [comment]
I think you could at least change `master` in the URL to a commit hash to lock the file to a specific commit, e.g., <https://raw.githubusercontent.com/publicsuffix/list/d6e0e8a8685495db53d2a8b0e3e6fbea4cdc2e76/public_suffix_list.dat>. No idea if that's the best way to do it with Portage, but I think it would fix the issue. :)

**Comment by TheJJ** [comment]
Yea, that's apparently the only way, they don't version otherwise i fear :)

**Comment by TheJJ** [comment]
fixed in d3bfa1a71e74f0a58bb9991339726f6f207aad3a, thanks for the hint!
